src/imgseg/segmentationUtils.py

```diff
@@ -13,12 +13,11 @@
 from scipy.ndimage.morphology import binary_fill_holes
 from scipy import ndimage
 from scipy import signal
-from scipy.misc import imsave
+from imageio import imsave
 
 import palettable   # pip install palettable
 
-# IMPORTS to create polygons from segmentation masks
-
+# IMPORTS to create polygons from label images
 from shapely.geometry import Polygon as shapely_polygon  # Used to simplify the mask polygons
 from descartes import PolygonPatch  # To plot polygons as patches (https://bitbucket.org/sgillies/descartes/src): pip install descartes
```

Comment:
If you introduce a new package, please make sure you add it into `requirements`.